app/components/isp-consent-form/consentText.js

```diff
@@ -0,0 +1,12 @@
+export default {
+    "test": {
+        "buttonLabel": "Please accept terms to continue",
+        "consentLabel": "I have read and understand the above statements and agree to participate.",
+        "paragraphs": {
```

Comment:
I should have noticed this sooner, Is there a benefit to having this be an Object and not and Array? JS Objects are _unordered_ sets of key-value pairs. Using this with `each-in` will not produce a reliable ordering.